one_fm/__init__.py

```diff
@@ -21,10 +21,13 @@
 from frappe.desk.doctype.notification_log.notification_log import NotificationLog
 from one_fm.api.notification import after_insert
 from one_fm.one_fm.payroll_utils import add_tax_components
+from hrms.overrides.employee_master import EmployeeMaster,validate_onboarding_process
+from one_fm.overrides.employee import EmployeeOverride
 
 __version__ = '14.0.0'
 
-
+EmployeeMaster.validate = EmployeeOverride.validate
+EmployeeMaster.validate_onboarding_process = validate_onboarding_process
```

Comment:
@Talleyrand333 Update the  usage??

Reply:
import `validate_onboarding_process` from EmployeeMaster
and then override `EmployeeMaster.validate_onboarding_process` with the imported `validate_onboarding_process`. Those are same!!